build/images/codegen/README.md

```diff
@@ -16,10 +16,13 @@ docker push antrea/codegen:<TAG>
 The `docker push` command will fail if you do not have permission to push to the
 `antrea` Dockerhub repository.
 
+The image can only be built on an x86_64 machine (no arm support).
```

Comment:
```suggestion
The image can only be built on a x86_64 machine (no arm support).
```

Reply:
Guess this is correct. x86's pronunciation starts with a vowel.